presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1676,7 +1676,7 @@ else if (column.isHidden()) {
         List<Column> tableColumns = table.getDataColumns();
         ImmutableMap.Builder<String, Optional<String>> builder = ImmutableMap.builder();
         for (Column field : concat(tableColumns, table.getPartitionColumns())) {
-            if ((field.getComment() != null) && !Optional.of("from deserializer").equals(field.getComment())) {
+            if ((field.getComment().isPresent()) && !Optional.of("from deserializer").equals(field.getComment())) {
```

Comment:
- rm unnecessary braces `(field.getComment().isPresent())`
- now you can change rhs of `&&` to `! field.getComment().get().equals("from deserializer")` which might be less tricky